<ui version="4.0" >
 <class>SetupDialogClass</class>
 <widget class="QDialog" name="SetupDialogClass" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>506</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="sizeConstraint" >
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="styledFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <property name="topMargin" >
         <number>5</number>
        </property>
        <property name="rightMargin" >
         <number>5</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>10</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="closeToolButton" >
          <property name="focusPolicy" >
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="accessibleName" >
           <string>transparentButton</string>
          </property>
          <property name="layoutDirection" >
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="popupMode" >
           <enum>QToolButton::DelayedPopup</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <property name="leftMargin" >
         <number>1</number>
        </property>
        <property name="topMargin" >
         <number>9</number>
        </property>
        <property name="rightMargin" >
         <number>1</number>
        </property>
        <property name="bottomMargin" >
         <number>6</number>
        </property>
        <item>
         <widget class="QTabWidget" name="setupTabWidget" >
          <property name="tabShape" >
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="profileTab" >
           <attribute name="title" >
            <string>ProfileTab</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5" >
            <item>
             <widget class="MoodBox::ProfileFrame" name="profileFrame" >
              <property name="frameShape" >
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth" >
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="settingsTab" >
           <attribute name="title" >
            <string>SettingsTab</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4" >
            <item>
             <widget class="MoodBox::SettingsFrame" name="settingsFrame" >
              <property name="frameShape" >
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth" >
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameBottom" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>16</width>
            <height>46</height>
           </size>
          </property>
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth" >
           <number>0</number>
          </property>
          <layout class="QHBoxLayout" name="_4" >
           <property name="rightMargin" >
            <number>16</number>
           </property>
           <property name="bottomMargin" >
            <number>5</number>
           </property>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="okButton" >
             <property name="minimumSize" >
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="accessibleName" >
              <string>rectButton</string>
             </property>
             <property name="text" >
              <string>OkButton</string>
             </property>
             <property name="autoDefault" >
              <bool>false</bool>
             </property>
             <property name="default" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancelButton" >
             <property name="minimumSize" >
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="accessibleName" >
              <string>rectButton</string>
             </property>
             <property name="text" >
              <string>CancelButton</string>
             </property>
             <property name="autoDefault" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <customwidgets>
  <customwidget>
   <class>MoodBox::ProfileFrame</class>
   <extends>QFrame</extends>
   <header>profileframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MoodBox::SettingsFrame</class>
   <extends>QFrame</extends>
   <header>settingsframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
